decidim/decidim
# Strong passwords for admins

#### :tophat: What? Why?
Adds configurable option to require strong passwords for admins. When option is set to true admins are asked to update their passwords to strong passwords after they log in.

**NOTE: If you are using seeds admin's default password has changed to**  ```decidim123456789```

#### :pushpin: Related Issues
Fixes #9178

#### Testing
1. ```Decidim::User.find(1).update(password_updated_at: nil)```
2. Login with user with id 1 ( admin@example.org )
3. See "Change your password" view

### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/169816942-751d8553-39e3-4e06-b71c-f9a27ae0ccdb.png)

:hearts: Thank you!
